STANDARD.md

```diff
@@ -12,9 +12,9 @@ interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
 
 All `keywords` in this standard are case-sensitive.
 
-The naming *(float)* without a closer specification is used if the implementor
+The naming *(floatX)* without a closer specification is used if the implementor
```

Comment:
I think the word *closer* is ambiguous here. I guess you mean `without any further specification`. But this applied to `float` (which did not have any further specification, like `float16`). With the new notation `floatX`, I am not sure the phrase `without a closer specification` makes sense anymore, it the sense that `X` is, in a way, a specification.

Reply:
That's a great point, we could maybe even shorten it to just `without further specification`?